gybitg/Controllers/SearchController.cs

```diff
@@ -89,79 +85,38 @@ public async Task<IActionResult> SearchResults(SearchViewModel SearchParam)
             string SearchPosition = SearchParam.Position.ToString();
             DateTime SearchGraduation = SearchParam.HSGraduationDate;
 
-            decimal SearchMaxPPG = SearchParam.MaxPPG;
-            decimal SearchMinPPG = SearchParam.MinPPG;
-
-            decimal SearchMaxMPG = SearchParam.MaxMPG;
-            decimal SearchMinMPG = SearchParam.MinMPG;
-
-            decimal SearchMaxTPMG = SearchParam.MaxTPMG;
-            decimal SearchMinTPMG = SearchParam.MinTPMG;
-
-            decimal SearchMaxFTMG = SearchParam.MaxFTMG;
-            decimal SearchMinFTMG = SearchParam.MinFTMG;
+            string SearchHS = SearchParam.HighSchool;
+            string SearchAAU = SearchParam.AAUId;
```

Comment:
I had not added info for AAUId for the repo athletes, I will do that asap if that is something that will be searchable.  Sorry.

Reply:
Sounds good